### ISSUE TITLE
Document the MDai algorithm

### ISSUE THREAD

**Issue by reubano** [concept-to-clinic/concept-to-clinic#24]
## Overview

Participants in the [Data Science Bowl](https://www.kaggle.com/c/data-science-bowl-2017) produced several algorithms that we would like to incorporate. To help facilitate this effort, we also want to add documentation so that contributors can make an educated decision when selecting an algorithm to incorporate.

## Expected Behavior

This documentation should enable people to:
 - view the library dependencies and license
 - understand its pros/cons
 - evaluate its performance/accuracy
 - identify which areas of the codebase to target for improvement

Design doc reference: [Detect and select](https://concept-to-clinic.readthedocs.io/en/latest/design-doc.html#detect-and-select)

## Algorithm info

key | value
----|------
team|MDai
rank|6
repo|https://github.com/mdai/kaggle-lung-cancer
trained models|requested
converted branch|
ML engine|Keras
engine-version|1.2.2
ML backend|Tensorflow
backend-version|1.0.0
training method|
architecture|
algorithm|
OS|Ubuntu
OS version|16.04
Python version|3.5
CUDA version|8
cuDNN version|5.1
notes|https://github.com/pydicom/pydicom/tree/bbaa74e9d02596afc03b924fe8ffbe7b95b6ff55

## Technical details

 - This feature should be implemented as a new markdown file in the [`docs folder`](https://github.com/concept-to-clinic/concept-to-clinic/tree/master/docs)

## Acceptance criteria

 - [ ] effective documentation for the above

*NOTE: All PRs must follow the standard PR checklist.*
